iam/iam-policy-for-cross-account-access/README.md

```diff
@@ -0,0 +1,30 @@
+# Terraform Module: IAM Policy for Cross Account Access
+
+By setting up cross-account access in this way, user don't have to create individual IAM users in each account.
+In addition, users don't have to sign out of one account and sign in to another account to access resources in different AWS accounts.
+
+## Usage
+
+### In destination account
+```terraform
+module "iam_cross_account_policies" {
+  source = "github.com/dbl-works/terraform//iam/iam-policy-for-cross-account-access/destination-account?ref=main"
+
+  bastion_aws_account_id = "xxxxxxxxxxxxxx"
+  policy_name = "AdministratorAccess"
```

Comment:
Right now, we have different access levels per person, e.g.:
person P1:
* staging access account A1 (via taggeable resources)
* production + staging access A2

person P2:
* admin access account A1
* admin access account A2

person P3:
* admin access account A1
* staging/production access A2

Is the cross-account access meant to be only for admins? If not, can we somehow reflect multiple roles for the same person across accounts?